e2e/token-vault-proxy/src/main.ts

```diff
@@ -3,16 +3,15 @@ import { proxy } from '@forgerock/token-vault';
 // Initialize the token vault proxy
 proxy({
```

Comment:
This is the proxy (iframe) initialization call, passing in the configuration to setup it's connection with the main app and the ForgeRock platform.